### PR TITLE
[release-ocm-2.6] Bug 2112803: Fix image references for the release branch so that they correctly point to :ocm-2.6 instead of :latest

### PR DESCRIPTION
**Pull request by eliorerz**
/cc @osherdp

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -111,7 +111,7 @@ endif
 # Operator Vars - these must be kept up to date
 BUNDLE_CHANNELS ?= alpha,ocm-2.6
 BUNDLE_OUTPUT_DIR ?= deploy/olm-catalog
-BUNDLE_METADATA_OPTS ?= --channels=$(BUNDLE_CHANNELS) --default-channel=alpha
+BUNDLE_METADATA_OPTS ?= --channels=$(BUNDLE_CHANNELS) --default-channel=ocm-2.6kustomization.yaml
```
Comment (osherdp) [inline Makefile:114]: probably pasted by mistake

**Review by osherdp — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (eliorerz) [reply Makefile]: Yep, sorry for that

**Review by osherdp — APPROVED** [review]

**Review by osherdp — APPROVED** [review]

**Comment by osherdp** [comment]
/retest
capi should be fixed, disconnected might also be affected by some other change

**Comment by osherdp** [comment]
Now that https://github.com/openshift/assisted-service/pull/4238 has been merged we should get a new result
/retest

**Comment by osherdp** [comment]
It gets us past operator installation now, so let's merge it
/override ci/prow/e2e-ai-operator-ztp-disconnected

**Pull request merged**